agreement/abstractions.go

```diff
@@ -234,6 +234,7 @@ type KeyManager interface {
 // MessageHandle is an ID referring to a specific message.
 //
 // A MessageHandle of nil denotes that a message is "sourceless".
+//msgp:ignore MessageHandle
```

Comment:
Yeah this was actually never necessary since ignore only prevents methods from being generated not from trying to serialize them if they are already required in the struct being serialized. This would only work if we could effectively make MessageHandle implement (Un)Marshaller interface(s).